easybuild/easyconfigs/p/pBWA/pBWA-0.5.9_1.21009-goolf-1.4.10.eb

```diff
@@ -0,0 +1,38 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2013-2014 Cyprus Institute
+# Authors::   Thekla Loizou <t.loizou@cyi.ac.cy>
+# License::   MIT/GPL
+# $Id$
+#
+##
+easyblock = "MakeCp"
+
+name = 'pBWA'
+version = '0.5.9_1.21009'
+
+homepage = 'http://pbwa.sourceforge.net/'
+description = """pBWA is a parallel implementation of the popular software BWA. It was developed by modifying the BWA source code with the OpenMPI C library on the SHARCNET. pBWA has been successfully tested on other systems with the most basic OpenMPI installs. pBWA currently implements three commands from BWA: aln, samse, and sampe. pBWA retains and improves upon the multithreading provided by BWA while adding efficient parallelization for the above listed functions. pBWA has shown that its wall-time speedup is bounded only by the size of the parallel system available as pBWA can run on any number of nodes and/or cores simultaneously."""
```

Comment:
please wrap into multiple lines, for readability!

also, trim redundant whitespace visible below (ie. extra lines, make dependencies more compact)